modules/opm-cli-ref-serve.adoc

```diff
@@ -5,7 +5,7 @@
 [id="opm-cli-ref-server_{context}"]
 = serve
 
-Serve declarative configs via a GRPC server.
+This command serve declarative configs via a GRPC server.
```

Comment:
Better to remove "This command", just "Serve declarative configs via a GRPC server." will be OK